src/layers/core/scatterplot-layer/scatterplot-layer-fragment.glsl

```diff
@@ -24,8 +24,19 @@
 precision highp float;
 #endif
 
+uniform float drawOutline;
+
 varying vec4 vColor;
+varying vec2 uv;
+varying float strokeWidthRatio;
 
 void main(void) {
-  gl_FragColor = vColor;
+
+  float distToCenter = length(uv);
+
+  if (distToCenter <= 1.0 && (drawOutline == 0.0 || distToCenter >= 1.0 - strokeWidthRatio)) {
```

Comment:
Instead of a `strokeWidthRatio` varying, maybe just precalculate `innerUnitRadius` defaulting to `0` and  do 
```
if (distToCenter <= 1.0 && distToCenter >= innerUnitRadius) {
```
A little more self documenting and possibly a tad more performant.